src/commands/chain/height.ts

```diff
@@ -35,12 +36,12 @@ export default class extends ProfileCommand {
 
         this.spinner.start();
         const chainHttp = profile.repositoryFactory.createChainRepository();
-        chainHttp.getBlockchainHeight().subscribe(
```

Comment:
This command displays the **chain height**, not specifically chain info.

I agree a command could be added for reading chain info and by that obsolete the chain height command. But for this PR I do not think it falls in scope as this only intends to upgrade CLI to network v.0.10.x compat.

Would love to see a new issue added for this. With latest features added, probably CLI can need a catch up.

Reply:
I did a mix of both here as a compromise because I know of some people that would want the chain height and/or score to be returned in raw format ... 

So what I did is in fact to satisfy your comment above with a new `chain info` command that will display the info in a table. 

There is a second table displaying finalization information underneath the basic block and height.

Thanks for pointing at the missing finalization info :+1: